lib/rules/no-deprecated.ts

```diff
@@ -38,6 +46,9 @@ export default createRule("no-deprecated", {
         if (typeof name !== "string" || typeof ver !== "string" || !ver) {
           return;
         }
+        if (allows.includes(name)) {
```

Comment:
Why not just:

```suggestion
        if (context.options[0]?.allows?.includes(name)) {
```

Reply:
Thank you for your review! I have changed it.